### PR TITLE
Fixes #16763: same as parent ticket but for postinst

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/16763

### CODE REVIEW

File: rudder-webapp/debian/preinst [hunk 1]
```diff
@@ -46,10 +46,8 @@ case "$1" in
         /etc/init.d/rudder-jetty stop >&2 > /dev/null || true
       fi
 
-      if [ -x /opt/rudder/bin/rudder-pkg ]
-      then
-        /opt/rudder/bin/rudder-pkg plugin save-status > /tmp/rudder-plugins-upgrade
-      fi
+      rudder package plugin save-status > /tmp/rudder-plugins-upgrade
```
Comment (amousset) [inline rudder-webapp/debian/preinst]: I'm not sure it will be there in preinst when upgrading from 5.0

Comment (ncharles) [reply rudder-webapp/debian/preinst]: ha yes

**Comment by ncharles** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**